install.php

```diff
@@ -121,7 +121,13 @@
 		$config = str_replace("##user##", $_POST['dbuser'], $config);
 		$config = str_replace("##password##", $_POST['dbpassword'], $config);
 		$config = str_replace("##name##", $_POST['servername'], $config);
+		$config = str_replace("##policy_name##", $POST['policy_name'], $config);
```

Comment:
Typo: should be _POST

Reply:
Ahhhh finally, Thank you... Searched houres for this typo...